### PR TITLE
Added Turkish translation

### PR DESCRIPTION
**Pull request by symbuzzer**

### CODE REVIEW

File: orion-viewer/src/main/res/values-tr/langs.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <string name="lang_english" translatable="false">en</string>
+        <string name="lang_russian" translatable="false">ru</string>
+        <string name="lang_hebrew" translatable="false">iw</string>
+        <string name="lang_chinese" translatable="false">zh</string>
+        <string name="lang_italian" translatable="false">it</string>
+        <string name="lang_spanish" translatable="false">es</string>
+        <string name="lang_ukrainian" translatable="false">ua</string>
+        <string name="lang_german" translatable="false">de</string>
+        <string name="lang_french" translatable="false">fr</string>
+        <string name="lang_turkish" translatable="false">tr</string>
```
Comment (max-kammerer) [inline orion-viewer/src/main/res/values-tr/langs.xml]: @symbuzzer Please remove this commit, just add 
`<string name="lang_turkish" translatable="false">tr</string>`
and 
`    <string name="lang_turkish_user">Turkish</string>`
to `orion-viewer/src/main/res/values/langs.xml`

Comment (max-kammerer) [reply orion-viewer/src/main/res/values-tr/langs.xml]: Please also update `orion-viewer/src/main/res/values/arrays.xml` file - `pref_languages` and `pref_languages_desc` arrays - with new translation

Comment (symbuzzer) [reply orion-viewer/src/main/res/values-tr/langs.xml]: > @symbuzzer Please remove this commit, just add `<string name="lang_turkish" translatable="false">tr</string>` and ` <string name="lang_turkish_user">Turkish</string>` to `orion-viewer/src/main/res/values/langs.xml`

It is already done?
image

Comment (symbuzzer) [reply orion-viewer/src/main/res/values-tr/langs.xml]: > Please also update `orion-viewer/src/main/res/values/arrays.xml` file - `pref_languages` and `pref_languages_desc` arrays - with new translation

Done

Comment (max-kammerer) [reply orion-viewer/src/main/res/values-tr/langs.xml]: >It is already done?

@symbuzzer No, I see that both commits  updating 'values-tr/langs.xml' are still present in pull request, they are redundant. Please just add
`<string name="lang_turkish" translatable="false">tr</string>`
and
` <string name="lang_turkish_user">Turkish</string>`
to default resource `.../values/langs.xml`

Comment (symbuzzer) [reply orion-viewer/src/main/res/values-tr/langs.xml]: Sorry for misunderstand. Done

**Comment by max-kammerer** [comment]
@symbuzzer Thank you! Merged manually

**Comment by max-kammerer** [comment]
@symbuzzer I forgot to ask you: usually I mention translators in [contributors section](https://github.com/max-kammerer/orion-viewer/blob/c13778519428ef43df76a9fbd56058a192a535d8/orion-viewer/src/main/res/layout/app_about_fragment.xml#L76) and in [translations wiki](https://github.com/max-kammerer/orion-viewer/wiki/Translations)

Can I mention you? If yes, how can I name you?

**Comment by symbuzzer** [comment]
Yes u can. My name is Ali Beyaz

**Pull request closed without merging**